packages/jest-console/src/__tests__/CustomConsole.test.ts

```diff
@@ -5,47 +5,85 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import {Writable} from 'stream';
 import chalk from 'chalk';
 import CustomConsole from '../CustomConsole';
 
 describe('CustomConsole', () => {
   let _console;
-  let _stdout = '';
+  let _stdout;
+  let _stderr;
 
   beforeEach(() => {
-    _console = new CustomConsole(process.stdout, process.stderr);
-    jest.spyOn(_console, '_logToParentConsole').mockImplementation(message => {
-      _stdout += message + '\n';
+    _stdout = '';
+    _stderr = '';
+
+    const stdout = new Writable({
+      write(chunk, encoding, callback) {
+        _stdout += chunk.toString();
+        callback();
+      },
     });
 
-    _stdout = '';
+    const stderr = new Writable({
+      write(chunk, encoding, callback) {
+        _stderr += chunk.toString();
+        callback();
+      },
+    });
+
+    _console = new CustomConsole(stdout, stderr);
+  });
+
+  describe('log', () => {
+    test('should print to stdout', () => {
+      _console.log('Hello world!');
+
+      expect(_stdout).toBe('Hello world!\n');
+    });
+  });
+
+  describe('error', () => {
+    test('should print to stderr', () => {
+      _console.error('Found some error!');
+
+      expect(_stderr).toBe('Found some error!\n');
+    });
+  });
+
+  describe('warn', () => {
+    test('should print to stderr', () => {
+      _console.warn('Found some warning!');
+
+      expect(_stderr).toBe('Found some warning!\n');
+    });
   });
 
   describe('assert', () => {
     test('do not log when the assertion is truthy', () => {
       _console.assert(true);
 
-      expect(_stdout).toMatch('');
+      expect(_stderr).toMatch('');
     });
 
     test('do not log when the assertion is truthy and there is a message', () => {
       _console.assert(true, 'ok');
 
-      expect(_stdout).toMatch('');
+      expect(_stderr).toMatch('');
     });
 
     test('log the assertion error when the assertion is falsy', () => {
       _console.assert(false);
 
-      expect(_stdout).toMatch('AssertionError');
-      expect(_stdout).toMatch('false == true');
+      expect(_stderr).toMatch('AssertionError');
+      expect(_stderr).toMatch('The expression evaluated to a falsy value:');
```

Comment:
Node 11 tries to be smart printing the line that threw the error but it doesn't work well.